src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -34,6 +34,11 @@
             <argument type="service" id="validator" />
         </service>
 
+        <service id="messenger.middleware.record_messages" class="Symfony\Component\Messenger\Middleware\HandleRecordedMessageMiddleware" abstract="true">
+            <argument type="service" id="message_bus" />
```

Comment:
Probably not a big deal, but I would use `<argument type="service" /> <!-- Message bus -->` to make it fully abstract

Reply:
Yes, But if you only have one bus configured, do you really want to explicit define it again in the config? Or maybe that is a good idea so the user understands what is happening.